src/backend/function/Fusion.Resources.Functions/local.settings.template.json

```diff
@@ -8,13 +8,13 @@
 
     "AzureWebJobsServiceBus": "[REPLACE WITH SB CONNECTION STRING]",
 
-    "AzureAd:TenantId": "3aa4a235-b6e2-48d5-9195-7fcf05b459b0",
-    "AzureAd:ClientId": "5a842df8-3238-415d-b168-9f16a6a6031b",
-    "AzureAd:Secret": "[REPLACE WITH SECRET]",
+    "AzureAd_TenantId": "3aa4a235-b6e2-48d5-9195-7fcf05b459b0",
```

Comment:
It is said that the __ should replace the :. We tried it in query, but maybe that was due to 3.0, not 3.1?

Reply:
Yeah, I've successfully used double underscore to replace colon in app service running on container, so in 3.1 it works for app service. Function might be a different story. We can try it out. If so we need to change code and ARM templates as well. I simply replaced stuff to make it run locally, which it does. Propose to do this in separate PR.

Reply:
Yeah should definitely a candidate for another pr..